control-plane/agents/Cargo.toml

```diff
@@ -17,6 +17,10 @@ path = "jsongrpc/src/server.rs"
 name = "common"
 path = "common/src/lib.rs"
 
+[[bin]]
+ name = "ha-node"
```

Comment:
We should start naming things in accordance to the new nomenclature? @paulyoong 
So the name here would be "agent-ha-node" ?

Reply:
Please rename this to "agent-ha-node".